python/src/aiconfig/editor/client/src/components/EditorContainer.tsx

```diff
@@ -594,14 +617,53 @@ export default function EditorContainer({
                 output: event.data,
               });
             } else if (event.type === "aiconfig") {
+              dispatch({
+                type: "CONSOLIDATE_AICONFIG",
+                action: {
+                  ...action,
+                  // Ensure we keep the prompt in a running state since this is an in-progress update
+                  isRunning: true,
+                },
+                config: event.data,
+              });
+            } else if (event.type === "aiconfig_complete") {
               dispatch({
                 type: "CONSOLIDATE_AICONFIG",
                 action,
                 config: event.data,
               });
             }
           },
-          enableStreaming
+          (event) => {
+            console.log(
```

Comment:
Maybe I should dispatch a run prompt error in this case where it isn't a cancellation? Currently we aren't propagating errors from the aiconfig.run call to here as far as I can tell

Reply:
Will do at top of stack. Currently no error is propagated other than cancellation anyways